### PR TITLE
pyDKB/InputStream: update descriptions.

### PR DESCRIPTION
**Pull request by Evildoor**
Initial idea was to merely fix a "retrun", but since it's a typo in a description that will be, sooner or latter, rewritten in a proper form, that "sooner or later" may as well be "now".

### CODE REVIEW

File: Utils/Dataflow/pyDKB/dataflow/communication/stream/InputStream.py [hunk 1]
```diff
@@ -38,6 +38,16 @@ def reset(self, fd, close=True, force=False):
         """ Reset current stream with new file descriptor.
 
         Overrides parent method to reset __iterator property.
+
+        :param fd: new file descriptor
+        :type fd: file
+        :param close: if True, close the old file descriptor
+        :type close: bool
+        :param force: if True, force the reset of iterator
```
Comment (mgolosova) [inline Utils/Dataflow/pyDKB/dataflow/communication/stream/InputStream.py:46]: I think it'd be nice to mention the difference between "forced" and "normal" reset in the description. Kind of this:

```
        :param force: if True, force the reset of iterator
                      (normally, it won't be reset if the passed new
                       descriptor is the same as the old one)
```

Comment (Evildoor) [reply Utils/Dataflow/pyDKB/dataflow/communication/stream/InputStream.py:46]: Done.

**Comment by Evildoor** [comment]
Force-push because I forgot to rebuild the docs.

**Comment by mgolosova** [comment]
@Evildoor, thank you, merged.

*Please remove the branch if no longer needed.*

**Pull request merged**